src/cdk/v2/destinations/smartly/config.js

```diff
@@ -0,0 +1,21 @@
+const { getMappingConfig } = require('../../../../v0/util');
+
+const ConfigCategories = {
+  TRACK: {
+    type: 'track',
+    name: 'trackMapping',
+  },
+};
+
+const mappingConfig = getMappingConfig(ConfigCategories, __dirname);
+const singleEventEndpoint = 'https://s2s.smartly.io/events';
```

Comment:
why are we not using this endpoint for processor ?

Reply:
Actually in integrations config and in rudder-transformer features.json, we have added configuration to let it go through router only.